Student/ruby_prac.rb

```diff
@@ -49,5 +49,4 @@ def to_s
 		average_phrase[-1] = "."
 		student + average_phrase
 	end
-
 end
```

Comment:
Here github is showing you the 🚫 icon because it's missing a line at the end. It's a bit of a silly thing but you can have your editor automatically add them for you.

If you go to [settings](https://stackoverflow.com/questions/20280159/change-default-settings-in-sublime-text-3) (Preferences -> Settings - User) and then add:

```json
"ensure_newline_at_eof_on_save": false,
```

It should be the last time you see the issue.

Other than that not sure if you meant to add more changes, it probably doesn't show anything else because all your other changes are already on master